packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -428,9 +425,12 @@ export class DependencyManager implements Disposable {
 export async function checkIfCLIInstalled(cmd: string, options: Record<string, unknown> = {}) {
   try {
     // We are not checking the stderr here, because some of the CLIs put the warnings there.
-    const { stdout } = await command(cmd, { encoding: "utf8", ...options });
-    return !!stdout.length;
+    const { stdout } = await command(cmd, { encoding: "utf8", quiet: true, ...options });
+    const result = !!stdout.length;
+    Logger.debug(`CLI: ${cmd} ${result ? "" : "not"} installed `);
```

Comment:
Nit: "CLI: " prefix is unnecessary.

Reply:
I think it is useful to have prefixes that allow to serach what part of our code generated the log. We have. a growing code base and most of that log is generated dynamically so searching for it would be hard without it.